Components/Modal Component/modal.css

```diff
@@ -0,0 +1,28 @@
+@import url('../../component.css');
+.modal-container{
+    position: absolute;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    width: 100vw;
+    display: none;
+    background-color: rgb(255 255 255 / 0.5);
+    top: 0;
+    left: 0;
+    z-index: 1;
```

Comment:
A small suggestion here would be that you could sort these properties in alphabetical order. As there are many properties, it might get difficult in debugging.

Reply:
i plan to do that before the final review since i've to do for all other components too